x-pack/plugins/rule_registry/server/plugin.ts

```diff
@@ -86,10 +119,44 @@ export class RuleRegistryPlugin
     });
 
     this.eventLogService = eventLogService;
-    return { ruleDataService, eventLogService };
+
+    return { ruleDataService: this.ruleDataService, eventLogService };
+  }
+
+  public start(core: CoreStart, plugins: RuleRegistryPluginStartDependencies) {
```

Comment:
Nit: may consider abstracting this inferred return type out to an explicit `RuleRegistryPluginStartContract` type.